hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - Overall Metrics.json

```diff
@@ -0,0 +1,2591 @@
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "repeat": "node_type",
+      "repeatDirection": "h",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus"
+          },
+          "exemplar": true,
+          "expr": "scm_node_manager_in_service_[[node_type]]_nodes",
+          "interval": "",
+          "legendFormat": "",
+          "refId": "A"
+        }
+      ],
+      "title": "in_service_[[node_type]]_nodes",
+      "type": "timeseries"
+    },
+    {
+      "collapsed": false,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 86
+      },
+      "id": 188,
+      "panels": [],
+      "title": "Datenode",
+      "type": "row"
+    },
+    {
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "palette-classic"
+          },
+          "custom": {
+            "axisLabel": "",
+            "axisPlacement": "auto",
+            "barAlignment": 0,
+            "drawStyle": "line",
+            "fillOpacity": 0,
+            "gradientMode": "none",
+            "hideFrom": {
+              "legend": false,
+              "tooltip": false,
+              "viz": false
+            },
+            "lineInterpolation": "linear",
+            "lineWidth": 1,
+            "pointSize": 5,
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "showPoints": "auto",
+            "spanNulls": false,
+            "stacking": {
+              "group": "A",
+              "mode": "none"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": [
+          {
+            "matcher": {
+              "id": "byFrameRefID",
+              "options": "A"
+            },
+            "properties": [
+              {
+                "id": "custom.axisPlacement",
+                "value": "left"
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 87
+      },
+      "id": 184,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom"
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus"
+          },
+          "exemplar": true,
+          "expr": "avg(container_cache_metrics_db_open_latency_avg_time{component=\"datanode\"})",
```

Comment:
This reads wrong, why do we need an avg for a metric reported as an avg.

Reply:
The raw thought is to make the overall graph more straightforward.  As there are a lot of DN DB instances in a normal cluster, we could detect the overall trend instead of each DB. Besides that here I put two metrics together with two Y-axises. 
<img width="1237" alt="QQ20220506-173226@2x" src="https://user-images.githubusercontent.com/10106574/167106379-81444cc9-da3e-4dc1-ab3c-e31182595b95.png">
Otherwise, it would be hard to distinguish the overall trend with too many instances. (
<img width="1257" alt="QQ20220506-173427@2x" src="https://user-images.githubusercontent.com/10106574/167107046-74540393-4343-4781-ad65-d7209941a4c3.png">
Users could add more variables to control the number of  exhibiting instances)